docs/source/markdown/options/attach.md

```diff
@@ -0,0 +1,10 @@
+#### **--attach**, **-a**=*stdin* | *stdout* | *stderr*
+
+Attach to STDIN, STDOUT or STDERR.
+
+In foreground mode (the default when **-d**
+is not specified), **podman run** can start the process in the container
+and attach the console to the process's standard input, output, and
+error. It can even pretend to be a TTY (this is what most command-line
+executables expect) and pass along signals. The **-a** option can be set for
+each of **stdin**, **stdout**, and **stderr**.
```

Comment:
The `podman-start` version, should anyone care to reconcile it:
```markdown
#### **--attach**, **-a**

Attach container's STDOUT and STDERR.  The default is false. This option cannot be used when
starting multiple containers.
```

Reply:
I don't see these as really an overlap.  I think they should stay separate.